pkgs/applications/science/math/gap/default.nix

```diff
@@ -141,12 +154,7 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "Computational discrete algebra system";
-    maintainers = with maintainers;
-    [
-      raskin
-      chrisjefferson
-      timokau
-    ];
+    maintainers = with maintainers; [ chrisjefferson ] ++ teams.sage.members;
     platforms = platforms.all;
     broken = stdenv.isDarwin;
```

Comment:
What is broken on Darwin / macOS?

Reply:
I don't know! This is preexisting, I will try to remove this line and see what CI says. Thanks for pointing this out!

Reply:
Turns out everything is working fine on Darwin. Nice catch, thanks!